plugins/nodes/src/js/columns/NodesTableMemBarColumn.tsx

```diff
@@ -3,12 +3,18 @@ import Node from "#SRC/js/structs/Node";
 // TODO: DCOS-39079
 // import { IWidthArgs as WidthArgs } from "@dcos/ui-kit/packages/table/components/Column";
 import { IWidthArgs as WidthArgs } from "#PLUGINS/nodes/src/js/types/IWidthArgs";
+import ProgressBar from "#SRC/js/components/ProgressBar";
 
 export function membarRenderer(data: Node): React.ReactNode {
-  // TODO: DCOS-38823
-  return <span>{data.get("used_resources").mem.toString()}</span>;
+  var value = data.get("used_resources").mem.toString();
+
+  return <ProgressBar
+  className="hidden-medium-down"
+  data={[{ value, className: 'color-2' }]}
```

Comment:
there was a small update to the issue, can you please change this to `color-1` to make it purple? 🙇